src/components/item-groups/link-item.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {GestureResponderEvent, TouchableOpacity, View} from 'react-native';
+import ThemeText from '../text';
+import NavigationIcon, {
+  isNavigationIcon,
+  NavigationIconTypes,
+} from '../theme-icon/navigation-icon';
+import useListStyle from './style';
+
+export type LinkItemProps = {
+  text: string;
+  subline?: string;
+  onPress?(event: GestureResponderEvent): void;
+  icon?: NavigationIconTypes | JSX.Element;
+};
+export default function LinkItem({
+  text,
+  onPress,
+  subline,
+  icon,
+}: LinkItemProps) {
+  const style = useListStyle();
+  const iconEl =
+    isNavigationIcon(icon) || !icon ? <NavigationIcon mode={icon} /> : icon;
+  return (
+    <TouchableOpacity onPress={onPress} style={style.baseItem}>
```

Comment:
Should  be provided an accessbility role prop. Perhaps link (by the judging the name of the component)?